### PR TITLE
Cards and Buttons

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**
Created Buttons and Cards 
Cards include: Standard Cards, Header Card, Horizonatal Card and Text Overlay card.
Buttons Include: Primary Buttons, Outline Buttons, Icon Button and floating action button.

### CODE REVIEW

File: component/Avatars/index.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+        <!-- ROUND TEXT AVATAR -->
+
+
+        <h4 class="avatar-subheading">> Round Text Avatar</h4>
+        <h5 class="avatar-content">Round Text Avatars shows the initals of your name in Round shape. As shown...</h5>
+<<<<<<< Updated upstream
+=======
+        
+>>>>>>> Stashed changes
```
Comment (gauravsinhaweb) [inline component/Avatars/index.html]: remove this text.

File: component/Avatars/index.html [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+</iframe>
+            </code>
+        </div>
+    </div>
+
+
+
+
+
+
```
Comment (gauravsinhaweb) [inline component/Avatars/index.html]: remove extra spacing.

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]
It would be more interactive if you give a hover effect on the navbar option.

The UI looks clean. Nice work!

File: component/Avatars/index.html (hunk 2 above)
Comment (gauravsinhaweb) [reply component/Avatars/index.html]: you can remove default margin in the website by ` html{ margin: 0; }` in your CSS.

File: component/Avatars/index.html (hunk 1 above)
Comment (vanshitamoolchandani) [reply component/Avatars/index.html]: That was a typo will remove it. Thanks.

File: component/Cards/index.html [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+                        <button class="btn-icon btn-icon-primary"><i class="fas fa-shopping-cart"></i></button>
+                        <button class="btn-icon btn-icon-primary"><i class="far fa-heart"></i></button>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+
+        <!-- HEADER CARD WITH BADGE -->
+        <div style="height: 30px;"></div>
```
Comment (AshutoshM29) [inline component/Cards/index.html]: Try avoiding inline-style
You can remove the div too I guess

File: component/Cards/index.html [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+        <h4 class="btn-subheading">> Header Card with Badge</h4>
```
Comment (AshutoshM29) [inline component/Cards/index.html:91]: Remove the extra ">"

File: component/Cards/index.html [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+                        <button class="btn-icon-child btn-icon-secondary"><i class="fas fa-shopping-cart"></i></button>
+                        <button class="btn-icon-child btn-icon-delete"><i class="far fa-heart"></i></button>
+                    </div>
+                </div>
+
+            </div>
+
+
+            <!-- HEADER CARD -->
+            <div style="height: 30px;"></div>
```
Comment (rahulk31) [inline component/Cards/index.html]: Plan to avoid inline styling

File: component/Cards/index.html [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+
+
+
+            <!-- HORIZONTAL IMAGE -->
+            <div style="height: 30px;"></div>
+
+            <div class="card-hi">
+                <div class="hi-align">
+                    <img class="hi-img" src="/images/headphones.jpg">
+
```
Comment (rahulk31) [inline component/Cards/index.html]: class names can be improvised.

File: component/Cards/styles.css [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+}
+
+
+/* BASIC END */
+
+/* HEADER WITH BADGE */
+.card-header-badge {
+    height: 480px;
+    width: 380px;
+}
```
Comment (rahulk31) [inline component/Cards/styles.css]: relative values are recommended.

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
For the outline delete button, you can give it some background color
Rest looks nice
You can merge it

**Review by rahulk31 — APPROVED** [review]
A small UI suggestion: Can decrease the width of the card so that every image can breath in.

image

Overall looks good to me. Liked that ribbon concept. 💯

File: component/Cards/index.html (hunk 4 above)
Comment (vanshitamoolchandani) [reply component/Cards/index.html:91]: Extra '>' was for defining a bullet point.

File: component/Cards/index.html (hunk 3 above)
Comment (vanshitamoolchandani) [reply component/Cards/index.html]: Will make the change. Thank you.

File: component/Cards/index.html (hunk 5 above)
Comment (vanshitamoolchandani) [reply component/Cards/index.html]: done. Thanks you

**Pull request merged**